libs/ledger-live-common/package.json

```diff
@@ -146,11 +146,11 @@
     "@ledgerhq/json-bignumber": "^1.1.0",
     "@ledgerhq/live-app-sdk": "^0.8.1",
     "@ledgerhq/logs": "workspace:^",
-    "@polkadot/types": "8.12.2",
-    "@polkadot/types-known": "8.12.2",
-    "@polkadot/util": "10.0.2",
-    "@polkadot/util-crypto": "10.0.2",
-    "@polkadot/wasm-crypto": "6.0.1",
+    "@polkadot/types": "9.4.2",
+    "@polkadot/types-known": "9.4.2",
+    "@polkadot/util": "10.1.9",
+    "@polkadot/util-crypto": "10.1.9",
+    "@polkadot/wasm-crypto": "6.3.1",
```

Comment:
There was an issue with wasm-crypto 6.2.3 unfortunately.
https://ledgerhq.atlassian.net/browse/LIVE-2976
I have little hope it works now, but let's test again...